### PR TITLE
KakaoEnterprise.KakaoWork version 1.7.0.2191

### PR DESCRIPTION
**Pull request by alexkoala**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19948)

### CODE REVIEW

File: manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KakaoEnterprise.KakaoWork
+PackageVersion: 1.7.0.2191
+PackageLocale: ko-KR
+Publisher: Kakao Enterprise
+PackageName: KakaoWork
```
Comment (ItzLevvie) [inline manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml:8]:
```suggestion
PackageName: Kakao Work
```

File: manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KakaoEnterprise.KakaoWork
+PackageVersion: 1.7.0.2191
+PackageLocale: ko-KR
+Publisher: Kakao Enterprise
```
Comment (ItzLevvie) [inline manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml:7]:
```suggestion
Publisher: Kakao Enterprise Corp
```

File: manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KakaoEnterprise.KakaoWork
+PackageVersion: 1.7.0.2191
+PackageLocale: ko-KR
+Publisher: Kakao Enterprise
+PackageName: KakaoWork
+License: Copyright ⓒ Kakao Enterprise. All rights reserved
+ShortDescription: KakaoWork Application
```
Comment (ItzLevvie) [inline manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml:10]:
```
ShortDescription: KakaoWork Application
```
Please write a better description.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml (hunk 3 above)
Comment (alexkoala) [reply manifests/k/KakaoEnterprise/KakaoWork/1.7.0.2191/KakaoEnterprise.KakaoWork.locale.ko-KR.yaml]: @ItzLevvie fixed

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**